### ISSUE TITLE
❓Proper error response on execute action endpoint?

### ISSUE THREAD

**Issue by luskin** [looker/actions#390]
Hello,

We have an internal, private action hub for our company that works very well. We have never been able to figure out, however, how to return a proper error so that the looker user receives the error email with context as to what was wrong. The error email always is as follows:

```
The scheduled job "Custom Action"[#####] failed for . Please visit your content to view and resolve the error, or contact your Looker administrator if you need further assistance.

Error: An error occurred for action test-action-integration://11::test_failure. The action could not be executed.
```

This is the error we are returning with a 400 back to looker:

```json
{
    "looker": {
        "success": false,
        "message": "Failed to update custom action",
        "validation_errors": {
            "test_field": "This is a test field error"
        }
    }
}
```

We are referencing the `ActionResponse` classes `toJson` schema [here](https://github.com/looker/actions/blob/82767e2170b11ec53bac4c37e9a16e3a43906fa4/src/hub/action_response.ts#L35)

Any help would be greatly appreciated. It's very difficult to scale our team using looker actions when an engineer has to hop into Datadog every time a team member gets one of these ambiguous errors.

**Comment by luskin** [comment]
Additionally, we have tried returning the JSON with both a 200 and a 400

**Comment by luskin** [comment]
Is anybody at Looker maintaining the `Action Hub`?

**Comment by phillipperalez** [comment]
Yes I still am maintaining the Action Hub, sorry I haven't  been posting on the issues lately

**Comment by phillipperalez** [comment]
So the email was a choice made a long time ago in favor of only showing the error in the scheduler history. It was a choice but we could open a ticket internally to start another discussion about it since it has been so long

**Comment by luskin** [comment]
It would be great to be able to pass a contextual error back to the user who scheduled the action

**Comment by hjorda** [comment]
Yes, I am having the same issue, it would be great to have a proper description of the error via e-mail. Any solution to this?
